job/mpijob/horovod-keras/job.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: batch.tensorstack.dev/v1beta1
+kind: MPIJob
+metadata:
+  name: keras-mnist-mpijob
+spec:
+  mca:
+    btl: ^openib
+    pml: ob1
+  mpiHome: /usr/local
+  ssh:
+    sshdPath: /usr/sbin/sshd
+  runPolicy:
+    cleanUpWorkers: true
+  worker:
+    replicas: 4
+    processesPerWorker: 1
+    processRecovery:
+      enable: true
+      limit: 100
+    cmd:
+      - python
+      - job/mpijob/horovod-keras/keras_mnist_mpijob.py
+      - "--log_dir"
+      - "job/mpijob/horovod-keras/log"
+      - "--no_cuda"
+    template:
+      spec:
+        containers:
+          - name: mpi-worker
+            image: registry.tensorstack.cn/t9k/horovod:20211201
+            imagePullPolicy: IfNotPresent
+            workingDir: /t9k/mnt/tutorial-examples
+            env:
+              - name: LD_LIBRARY_PATH
+                value: "/usr/local/cuda/compat/lib:/usr/local/nvidia/lib:/usr/local/nvidia/lib64:/usr/local/lib"
```

Comment:
Does this example use GPU? If yes, there should be extended resource requests, if not, these cuda/nvidia related paths are useless. 

env `NCCL_DEBUG` is for nvidia lib too, is it really required?

Reply:
removed

Reply:
so it's been verified that the sample works after removing these 2 env entries?